README.md

```diff
@@ -39,28 +39,26 @@ assert value == "hello world!"
 
 ## Development Setup
 
-0. Install Rust and Python
-1. Install `maturin` in order to build Ypy
-
+0. [Install Rust](https://www.rust-lang.org/tools/install) and [Python](https://www.python.org/downloads/) (consider [pyenv](https://github.com/pyenv/pyenv))
+1. Install [hatch](https://hatch.pypa.io/1.2/install/) (`python -m pip install hatch`)
+2. Create a development build of the library
+   
 ```
-pip install maturin
+python -m hatch run maturin develop
```

Comment:
I think it is a bit too much to always prepend `python -m`:
```suggestion
hatch run maturin develop
```

Reply:
Did you get a binary on your path when you installed hatch? I have Python 3.9.11 via pyenv as my default python (WSL OS). When I `python -m pip install hatch`, I can use `python -m hatch` afterwards but `hatch` is not found. I was going to actually drop a note on the hatch repo saying their docs seemed inconsistent with my experience.

Reply:
Yes, `hatch` is in my path. Maybe worth opening an issue there?

Reply:
yep I'll bring it up, switched the examples to use `hatch` and put a caveat next to the CONDA env caveat